### DISCUSSION TITLE
Allow user to choose which data to show in live timing window

### DISCUSSION THREAD

**Discussion by dickwolff** [f1multiviewer/issue-tracker#95] [Feature requests] [answered]
There is a lot of data on the screen, however not all data is relevant to me. For example during the race I don't care for sector times. 

It would be nice if you could select which data column to show, and also in which order.

**Comment by jeppeolesen** [comment, marked as answer]
Hey @dickwolff 

You can customise the live timing layouts by clicking the edit button on the top right.

![Screenshot 2025-03-22 at 23 11 33](https://github.com/user-attachments/assets/86c2ad1c-8fd8-4644-ab69-45a839480f49)
